src/main/java/org/cascadebot/cascadebot/data/objects/guild/GuildModeration.java

```diff
@@ -3,19 +3,21 @@
  * Licensed under the MIT license.
  */
 
-package org.cascadebot.cascadebot.data.objects;
+package org.cascadebot.cascadebot.data.objects.guild;
 
 import lombok.AccessLevel;
 import lombok.Getter;
 import lombok.NoArgsConstructor;
 import lombok.Setter;
 import org.cascadebot.cascadebot.commandmeta.Module;
+import org.cascadebot.cascadebot.data.objects.Setting;
+import org.cascadebot.cascadebot.data.objects.SettingsContainer;
 
 @SettingsContainer(module = Module.MODERATION)
 @Getter
 @Setter
 @NoArgsConstructor(access = AccessLevel.PUBLIC)
-public class GuildSettingsModeration {
+public class GuildModeration {
```

Comment:
This is named `GuildSettingsModeration` for convention sakes, I'd really vote to keep it that way